tests/test_video_manager.py

```diff
@@ -281,3 +282,19 @@ def test_multiple_videos(test_video_file):
         # Will release the VideoManagers in vm_list as well.
         video_manager.release()
 
+def test_many_videos_downscale_detect_scenes(test_video_file):
+    """ Test VideoManager handling decoding frames across video boundaries. """
+
+    NUM_FRAMES = 10
```

Comment:
Unnecessary constant, can probably be removed?

Reply:
Yeah, I'll leave that out in the next commit.